docs/user-guide/configuring.md

```diff
@@ -1085,3 +1079,23 @@ foo.js
 ```
 
 This message occurs because ESLint is unsure if you wanted to actually lint the file or not. As the message indicates, you can use `--no-ignore` to omit using the ignore rules.
+
+## Personal Configuration File (deprecated)
+
+⚠️ **This feature has been deprecated**. It will start to print deprecation warnings on ESLint `7.0.0` and will be removed on ESLint `8.0.0`. If you want to continue to use personal configuration file, please use [`--config` CLI option](https://eslint.org/docs/user-guide/command-line-interface#-c---config). If you want to know the discussion of this decision, see [RFC 28](https://github.com/eslint/rfcs/pull/28) and [RFC 32](https://github.com/eslint/rfcs/pull/32).
+
+Here `~/` means [the home directory of the current user on your OS](https://nodejs.org/api/os.html#os_os_homedir). The personal configuration file is `~/.eslintrc.*` file. ESLint handles `~/.eslintrc.*` file in special.
+
+### How ESLint Finds Personal Configuration File
+
+If `eslint` could not find any configuration file in the project, `eslint` loads `~/.eslintrc.*` file.
+
+If `eslint` could find configuration files in the project, `eslint` ignores `~/.eslintrc.*` file even if it's in an ancestor directory of the project directory.
+
+### How Personal Configuration File Behaves
+
+`~/.eslintrc.*` file behaves samely as regular configuration files, but there are some notes.
+
+The `~/.eslintrc.*` file loads shareable configs and custom parsers from `~/node_modules/` as similar to `require()` in home directory. Please mind that it doesn't load global-installed packages.
+
+The `~/.eslintrc.*` file loads plugins from `$CWD/node_modules` because ESLint loads plugins from `$CWD/node_modules` at all in order to identify plugins uniquely. If you want to use plugins with `~/.eslintrc.*` file, you have to install the plugins into each project. Or you can use [`--resolve-plugins-relative-to` CLI option](https://eslint.org/docs/user-guide/command-line-interface#--resolve-plugins-relative-to) to change the location where ESLint loads plugins from.
```

Comment:
```suggestion
The `~/.eslintrc.*` file loads plugins from `$CWD/node_modules` because ESLint loads plugins from `$CWD/node_modules` by default in order to identify plugins uniquely. If you want to use plugins with `~/.eslintrc.*` file, you have to install the plugins into each project. Or you can use [`--resolve-plugins-relative-to` CLI option](https://eslint.org/docs/user-guide/command-line-interface#--resolve-plugins-relative-to) to change the location where ESLint loads plugins from.
```